CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.36
+
+<!-- generated comparing v5.0.0-alpha.35..next -->
+
+_Jun 7, 2021_
+
+Big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support.
+
+`experimentalStyled` is now available without the `experimental` prefix.
+We're confident that it's API shouldn't receive any major changes until the stable release of v5.
+
+`@material-ui/icons` only ships ES modules and no longer CommonJS modules.
+This reduces the download size of the package from 1.7 MB to 1.2 MB and install size from 15.6 MB to 6.7 MB.
+
+### `@material-ui/core@5.0.0-alpha.36`
+
+#### Breaking changes
+
+- <!-- 10 --> [AppBar] Fix background color on dark mode (#26545) @siriwatknp
+- <!-- 13 --> [core] Rename `experimentalStyled` to `styled` (#26558)
+
+  Remove the experimental prefix, this module is going stable:
+
+  ```diff
+  -import { experimentalStyled as styled } from '@material-ui/core/styles';
+  +import { styled } from '@material-ui/core/styles';
+  ```
+
+- <!-- 03 --> [SwitchBase] replace IconButton with ButtonBase (#26460)
+- <!-- 25 --> [theme] Improve default `primary`, `secondary` and `error` theme palette (#26555) @siriwatknp
+
+#### Changes
+
+- <!-- 17 --> [Box] Fix module 'clsx' not found in system (#26553) @coder-freestyle
+- <!-- 07 --> [Box] Fix runtime error when using styled-components without ThemeProvider (#26548) @mnajdova
+- <!-- 27 --> [Radio][Checkbox] Don't forward `color` to DOM elements (#26625) @siriwatknp
```

Comment:
Maybe to provide a bit more context? (it's a regression)

```suggestion
- <!-- 27 --> [Radio][Checkbox] Fix forward `color` to DOM elements regression (#26625) @siriwatknp
```

Reply:
Why is that important?

Reply:
Ok, resolving